backup_and_restore/application_backup_and_restore/aws-sts/oadp-aws-sts.adoc

```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-aws-sts"]
+= Backing up applications on AWS STS using OADP
+include::_attributes/common-attributes.adoc[]
+:context: oadp-aws-sts-backing-up-applications
+
+toc::[]
+
+You install the OpenShift API for Data Protection (OADP) with Amazon Web Services (AWS) by installing the OADP Operator. The Operator installs link:https://{velero-domain}/docs/v{velero-version}/[Velero {velero-version}].
```

Comment:
Use the attribute for "OpenShift API for Data Protection (OADP)". Looks like the attribute is: {oadp-first}

Use the attribute for "Amazon Web Services (AWS)". Looks like that attribute is: {aws-first}

Use the attribute for "OADP". Looks like that attribute is: {oadp-short}

See: https://redhat-documentation.github.io/supplementary-style-guide/#_product_names_and_version_references 

Curious: should "OADP Operator" be its own attribute? Maybe something to consider.